async fn main() -> std::io::Result<i32, futures::future::err> {
    Ok(1)
}
